bin/dev_assets

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+"$(dirname "$0")"/dev -m js=1,css=1
```

Comment:
TIL
I didn't know about `foreman start`'s `-m` argument

Reply:
I didn't either. I spent some time looking up how to do this :stuck_out_tongue_closed_eyes: